samples/nrf9160/modem_shell/README.rst

```diff
@@ -933,6 +933,63 @@ For example:
 
    west build -p -b nrf9160dk_nrf9160_ns -- -DOVERLAY_CONFIG=overlay-non-offloading.conf
 
+BT shell support
+================
+
+To build the MoSh sample with Zephyr BT shell command support, use ``-DDTC_OVERLAY_FILE=bt.overlay`` and  ``-DOVERLAY_CONFIG=overlay-bt.conf`` options.
+When running this configuration, BT scanning and advertising can be mabe with ``bt`` command. Compilation for an example:
+
+.. code-block:: console
+
+   west build -p -b nrf9160dk_nrf9160_ns -- -DDTC_OVERLAY_FILE="bt.overlay" -DOVERLAY_CONFIG="overlay-bt.conf"
+
+Additionally, NRF52 side of the development kit needs to be programmed as instructed in :ref:`lte_ble_gateway`. Compilation of :ref:`uart_nrf_sw_lpuart`:
```

Comment:
The `n` in `nRF52` should be lower case.

Reply:
fixed